model/src/lectures/lectureDoc.ts

```diff
@@ -56,10 +62,14 @@ namespace LectureDisplay {
 		nbOfPages: number;
 		availableToStudents: boolean;
 		audioTranscript: { [pageNumber: number]: MultiLangTranscript; };
+		event?: LectureEventBase;
 	}
 
 	export const Functions = FunctionFolder("lectures", {
 		addAudioTranscript: FunctionOf<{ courseId: string, lectureId: string, pageNumber: number, transcription: string; }, {}, 'invalid_arg' | 'error_firebase' | 'successfully_added'>("addAudioTranscript"),
+		markQuestionAsAnswered: FunctionOf<{ id: string, courseId: string, lectureId: string, answered: boolean }, { id: string; }, 'invalid_id' | 'error_firebase' | 'empty_question'>("markQuestionAsAnswered"),
+		broadcastQuestion: FunctionOf<{ id: string, courseId: string, lectureId: string }, { id: string; }, 'invalid_id' | 'error_firebase' | 'empty_question'>("broadcastQuestion"),
+		clearQuestionEvent: FunctionOf<{ id: string, courseId: string, lectureId: string }, { id: string; }, 'invalid_id' | 'error_firebase' | 'empty_question'>("clearQuestionEvent"),
```

Comment:
You use multiple times some errors by exemple 'invalid_id', you can maybe put it in a constant.

Reply:
Alright, looking into it. The reason I did it this way, it because it the way we all were doing it in the project. This is indeed a problem of technical debt. Back on sprint one, we should have created a `ReturnedError` file containing those values. To stay consistent accross the entire codebase, I would prefer to stick to this fashion, as a matter of consistency. Hope you understand.